listenbrainz/listenstore/redis_listenstore.py

```diff
@@ -1,28 +1,31 @@
-# coding=utf-8
+from datetime import datetime
+from time import time
 
 import ujson
-import redis
-from time import time
-from redis import Redis
 from typing import Optional
 
 from listenbrainz.listen import Listen
 from listenbrainz.listenstore import ListenStore
-from datetime import datetime
+from brainzutils import cache
+from listenbrainz.utils import create_path, init_cache
 
 
 class RedisListenStore(ListenStore):
 
-    RECENT_LISTENS_KEY = "lb_recent_sorted"
+    RECENT_LISTENS_KEY = "rl-"
     RECENT_LISTENS_MAX = 100
+    PLAYING_NOW_KEY = "pn."
     LISTEN_COUNT_PER_DAY_EXPIRY_TIME = 3 * 24 * 60 * 60  # 3 days in seconds
-    LISTEN_COUNT_PER_DAY_KEY_FORMAT = "lb_listen_count_for_day_{}"
+    LISTEN_COUNT_PER_DAY_KEY_FORMAT = "lc-day-"
```

Comment:
I guess this was called `_FORMAT` because it contains a format string that we used `.format()` with. As you're just using it as a prefix with string concatenation, perhaps we could call it `LISTEN_COUNT_PER_DAY_KEY` or `LISTEN_COUNT_PER_DAY_KEY_PREFIX`?